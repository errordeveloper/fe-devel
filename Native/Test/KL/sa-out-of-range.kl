/*
 *  Copyright 2010-2012 Fabric Engine Inc. All rights reserved.
 */

operator entry() {
  String slicedArray<>( Size(16) );
  for( Size i=0; i<16; ++i )
    slicedArray[i] = i;
  report(slicedArray[7]);
  report(slicedArray[16]);
}
