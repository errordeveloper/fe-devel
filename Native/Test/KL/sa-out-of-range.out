7
Caught exception: sa-out-of-range.kl:10:10: index (16) out of range (16)
